src/TSMapEditor/UI/Windows/TriggersWindow.cs

```diff
@@ -1417,6 +1423,14 @@ private void BtnDeleteAction_LeftClick(object sender, EventArgs e)
             EditTrigger(editedTrigger);
         }
 
+        private void BtnCloneAction_LeftClick(object sender, EventArgs e)
+        {
+            if (editedTrigger == null || lbActions.SelectedItem == null)
+                return;
+
+            CloneEventOrAction(lbActions, editedTrigger.Actions);
+        }
```

Comment:
`CloneEventOrAction` already does verification on active trigger and list box selected item.

```suggestion
        private void BtnCloneAction_LeftClick(object sender, EventArgs e) => CloneEventOrAction(lbActions, editedTrigger.Actions);
```